pages/[resourceType]/create.tsx

```diff
@@ -49,6 +48,72 @@ const CreateResourcePage = () => {
       </Center>
     </div>
   );
+
+  //handles what will happen when the submit button is clicked.
+  function editorSubmitHandler() {
+    let notifProps = {
+      message: <div>Problem connecting to server</div>,
+      color: "red",
+      icon: <X size={18} />,
+      autoClose: false,
+    };
+
+    fetch(`${process.env.NEXT_PUBLIC_DEQM_SERVER}/${resourceType}`, {
+      method: "POST",
+      body: codeEditorContents,
+      headers: {
+        "Content-Type": "application/json+fhir",
+      },
+    })
+      .then((response) => {
+        let newID;
+
+        if (response.status === 201 || response.status === 200) {
+          newID = resourceType
+            ? response.headers.get("Location")?.substring(7 + resourceType.length)
```

Comment:
Hmm what might be even better is being more expressive about looking for resourceType/id in the location. We already know the resource type, and we know that it'll be a slash followed by some string of characters for the ID. can we use a regex for this?

Reply:
I created this regex: 
`const NEW_ID_IN_HEADER_REGEX = new RegExp(`${resourceType}/.{1,50}`);`
This is my first time using regex so there's probs a better way I could do it. Lmk if that's what we're going for whenever you go over my newest push.

Reply:
This looks solid! I can make one suggestion for a further iteration on the regex though:

We know that this part of the string (the one after the `resourceType/` must be a FHIR `id` type, which the FHIR spec actually defines a regex for: https://build.fhir.org/datatypes.html#id

So we can probably use this instead of `.{1,50}` for that part of the regex?